tst/CTA.Rules.Test/Actions/ActionHelpers/ConfigMigrateTests.cs

```diff
@@ -165,5 +165,129 @@ public void LoadWebConfigWithErrors()
 
             Assert.Null(doc);
         }
+
+        [Test]
+        public void MigrateServerConfig()
+        {
+            // Get private method to invoke
+            var projectType = ProjectType.Mvc;
+            ConfigMigrate configMigrateInstance = new ConfigMigrate(Directory.GetCurrentDirectory(), projectType);
+            Type configMigrateType = configMigrateInstance.GetType();
+
+
+            var loadWebConfigMethod = TestUtils.GetPrivateMethod(configMigrateType, "LoadWebConfig");
+            var portServerConfigMethod = TestUtils.GetPrivateMethod(configMigrateType, "PortServerConfig");
+
+            var webConfig = @"
```

Comment:
add more combinations to handle negative situations.. like no handlers tag... or no security tag... security with form auth... authorization with allow and deny roles or users... basically to make sure we don't throw errors or handle properly

Reply:
as of this release we are only working window authentication. I can add more auth types going forward

Reply:
@birojnayak will add test as part of this issue - https://github.com/aws/cta/issues/311